package.json

```diff
@@ -181,7 +182,7 @@
     "json-stringify-safe": "5.0.1",
     "lodash": "^4.17.21",
     "lru-cache": "^4.1.5",
-    "minimatch": "^3.0.4",
+    "minimatch": "3.0.5",
```

Comment:
let's try keep the caret `^` for package `minimatch`, this would update the package to all future minor/patch versions, without incrementing the major version. It's preferred to stay with a semver range instead of specific verion.

Reply:
yes.